### ISSUE TITLE
How do you get the final results?

### ISSUE THREAD

**Issue by devinxzhang** [nayeemrizve/OpenLDN#2]
Hello, thanks for your work!

When I read your code, I noticed that the final ACC was directly obtained by using the test set as the valid set. Would this be unfair?

Thanks!

**Comment by nayeemrizve** [comment]
Hi, thank you for expressing interest in our work. It's true that we haven't used a validation set for model selection. However, to compensate for that we generate pseudo-labels from the latest model from the first stage training and also pick the last model from the second stage (closed-world) training to report final accuracy. Besides, We haven't done extensive hyperparameter tuning. The only hyperparameter we did tune during development was the learning rate and that was tuned separately on a validation set created from the labeled subset. Hope this clarifies your concerns.
